apps/docs-website/docs/core-philosophy/criteria.md

```diff
@@ -5,16 +5,16 @@ description: 🎯 What line of reasoning drives Sheriff development
 
 # Criteria
 
-This library is very opinionated, but it's for the better. A lot of decisions were taken so you don't have to.<br />
-You can now quickstart static analysis in all your Typescript projects with ease. <br />
+This library is very opinionated, but for the better. A lot of decisions were taken so you don't have to.<br />
+You can now quickstart static analysis in all your TypeScript projects with ease.<br />
 Just type `npm init @sherifforg/config` in a CLI and you are good to go.
 
-Easiness of use without compromises is the top priority of Sheriff. The basic principle behind many design decisions of Sheriff is to require as less inputs from the user as possible. <br />
-You can think of Sheriff like Prettier or `create-react-app`. It's a tool that comes battery-packed with optimal defaults. It removes configuration decisions from the equation, so you or your team can focus on developing the actual product.<br />
-And if you don't like something, you can easily override it, and just as easily you can extend it. See: [configuration](../configuration.mdx).
+Ease of use without compromises is the top priority of Sheriff. The basic principle behind many design decisions of Sheriff is to require as less inputs from the user as possible.<br />
+You can think of Sheriff like Prettier. It’s a tool that comes battery-packed with optimal defaults. It removes configuration decisions from the equation, so you or your team can focus on developing the actual product.<br />
+If you disagree with something, you can easily override it, and you can extend it just as easily. See: [configuration](../configuration.mdx).
 
 This config is particularly useful for big teams with developers of various skill levels.<br />
-Sheriff was made to prevent all kind of mistakes and to align the team on the same playing field. It is battle-tested in real-world scenarios and shines especially in such.<br />
-Also, Sheriff target pretty much everybody. Junior developers can learn a lot by using it, and Lead developers can save a lot of time on configuration and maintenance.
+Sheriff was made to prevent all kind of mistakes and to align teams stylistically. It is battle-tested in real-world scenarios and shines especially in such.<br />
+Also, Sheriff targets pretty much everybody. Junior developers can learn a lot by using it, and lead developers can save a lot of time on configuration and maintenance.
```

Comment:
targets 👍
lead 👎

Reply:
Lead isn't a proper noun, it shouldn't be capitalized.

Reply:
lowercase can be confused with the verb. In this context it has the meaning of "Leaders"

Reply:
And uppercase can be confused with the verb as well... if you're concerned about confusing, perhaps use "leaders" or "team leads" or something else, but capitalizing it isn't a solution.

Reply:
But it is a used as a "company role" here. I think "Team Leads" should be the clearer option here